docs/Anjay_integrations/Anjay_Nordic_integration/Thingy91.md

```diff
@@ -62,8 +63,10 @@ After you have programmed and reset the board, go to the next step.
     stop    :Stop Anjay
     config  :Configure Anjay params
     ```
+In this situation really useful can be **Tab key** which shows you available possibilities.
+
+3. You can also change the default credentials to your custom data by following the instructions presented in the program.
 
-3. Change the default credentials to your custom data by following the instructions presented in the program and save it.
 
 ## Connect to the LwM2M Server
```

Comment:
I remember you said that you need to first add the device to the server, then provide credentials to the device, so you wanted to swap this section with `Configure the client`